content/2021-12-01-brigade-2-goes-ga.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Brigade 2 Goes GA"
+slug: "2021-12-01-brigade-2-goes-ga"
+description: "Brigade 2 is now generally available"
+date: "2021-12-01"
+author: "Kent Rancourt"
+summary: Brigade 2 is now generally available.
+---
+
+Alright, Brigadiers! This is the day we've all been waiting for. Brigade 2 is generally available! If you're ready to get started, head on over to our [QuickStart](https://docs.brigade.sh/intro/quickstart/).
+
+But that's not all!
+
+As promised in [a previous post](2021-11-17-brigade-v1-support-window), we've given some overdue love to Brigade 1.x as it enters a six month maintenance window. Yesterday, we released v1.9.0 of our Helm chart. It installs the existing Brigade v1.4.0 software, but addresses the deprecation or removal of certain beta APIs from newer releases of Kubernetes, making v1.4.0 installable, for the first time, on the newest, shiniest clusters. We've also released Brigade v1.5.0, which primarily remediates high severity vulnerabilities in the worker's JavaScript dependencies. Brigade v1.5.0 is installable using Helm chart v1.10.0.
```

Comment:
[a previous post](2021-11-17-brigade-v1-support-window)

This link is broken in https://deploy-preview-21--blog-brigade-sh.netlify.app//2021-12-01-brigade-2-goes-ga/ - probably needs a leading `/`?

Reply:
Would you like to link to the Helm charts and the releases that you mention? This paragraph is full of versions to mentally juggle. I almost wonder if it would be less-confusing as a table?

Reply:
Linking to a Helm chart feels a bit unnatural, no? That's not how you use it.

Reply:
(nods) I'm wondering what to link so that if someone thinks this section applies to them, they know where to go next. Maybe the 1.5 release notes?

Reply:
A table won't allow as easily for the color commentary-- like "making v1.4.0 installable, for the first time, on the newest, shiniest clusters."

I'm going to experiment with other formatting options though.